lib/v2/pts/index.js

```diff
@@ -38,11 +38,11 @@ module.exports = async (ctx) => {
 
                 const content = cheerio.load(detailResponse.data);
 
-                item.author = content('meta[name="author"]')
-                    .attr('content')
-                    .replace(/文／/, '')
-                    .split(/／|\/|編譯|報導/)[0];
-                item.pubDate = timezone(parseDate(content('meta[property="article:published_time"]').attr('content')), +8);
+                item.author = content('.reporter-container a')
+                    .toArray()
+                    .map((e) => content(e).text());
```

Comment:
`author` accepts either a plain string or an array of object that each object [contains `name`](https://github.com/DIYgod/RSSHub/blob/cdc36d3554789bce4d7e457906f4871f08c11a54/lib/middleware/template.js#L71)